style.css

```diff
@@ -1,624 +1,995 @@
+}
+
+#fifth-work {
+  background-image: linear-gradient(179.35deg, rgba(38, 38, 38, 0) 0.85%, rgba(38, 38, 38, 0.9) 84%), url("./images/works/first-image.png");
+}
+
+#sixth-work {
+  background-image: linear-gradient(179.35deg, rgba(38, 38, 38, 0) 0.85%, rgba(38, 38, 38, 0.9) 84%), url("./images/works/sec-image.png");
+}
+
+#seventh-work {
+  background-image: linear-gradient(179.35deg, rgba(38, 38, 38, 0) 0.85%, rgba(38, 38, 38, 0.9) 84%), url("./images/works/third-image.png");
+}
+
+.work-title {
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  color: #091e42;
+  margin: 5px 0 5px 0;
+}
+
+.works-description {
+  margin: 5px 0 5px 0;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #091e42;
+}
+
+#personal-data {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  max-width: 375px;
+  height: 1900px;
+  padding: 20px 20px;
+  background-color: #000;
+  margin: 0 auto;
+  overflow: hidden;
+}
+
+#my-cv-here {
+  background-image: url("./images/about_me_image.svg");
+  background-repeat: no-repeat;
+  background-position: right bottom;
+  height: 650px;
+  object-fit: contain;
+}
+
+.card-link {
+  text-decoration-line: none;
+  color: #fff;
+  background-color: #f55800;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  padding: 12px;
+  gap: 10px;
+  width: 171px;
+  height: 48px;
+}
+
+#my-cv-here a:active {
+  background: #e05304;
+}
+
+#my-cv-here a:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+#my-cv-here a:disabled {
+  background: #f4f1f0;
+  border: none;
+  color: #5b5957;
+}
+
+#my-cv-here a:enabled {
+  background: #f55800;
+  border: 1px solid #f55800;
+  color: #fff;
+}
+
+#my-cv-here h2 {
+  color: #f55800;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 82px;
+}
+
+#my-cv-here p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+#my-skills {
+  border-top: 1px solid #888;
+  clear: both;
+  text-align: center;
+  padding: 120px 0 0 0;
+}
+
+#my-skills ul {
+  font-size: 35px;
+  float: center;
+  text-indent: -30px;
+}
+
+ul.skill {
+  list-style: none;
+  font-size: 22px;
+}
+
+.contact-me {
+  max-width: 375px;
+  padding: 70px 20px 30px;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #fff;
+  overflow: hidden;
+}
+
+.contact-text {
+  padding-left: 130px;
+  width: 100%;
+  height: auto;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  justify-content: flex-end;
+  color: #262626;
+  text-align: left;
+  font-style: normal;
+  margin-top: -50px;
```

Comment:
- [ ] Please check out the `contact me` section styles on the mobile version and fix it because currently it looks like this 👇 
**Given design**
![image](https://user-images.githubusercontent.com/90222110/200013299-1a1e326c-ca24-45f9-bb46-af027a8da61e.png)

**Your design**
![image](https://user-images.githubusercontent.com/90222110/200013653-cf4f970b-299f-4757-bf76-a655f68431c9.png)

Reply:
![form-issue](https://user-images.githubusercontent.com/32777408/200088975-101c1fb2-5604-4097-9624-de607e1d8c2f.png)


I have resolved this issue. Thanks for all feedback raised. @BertrandConxy